### ISSUE TITLE
Internet Connectivity Setting

### ISSUE THREAD

**Issue by garkok** [goFrendiAsgard/No-CMS#169]
If Internet Connectivity setting is set to offline, No-CMS will not send any e-mail. For example: There is a check in Main.php/forgot that will query for this setting, and will fail to send mail.

**Comment by goFrendiAsgard** [comment]
I think if the server is not connected to the internet, we won't be able to send email. Isn't it?

**Comment by garkok** [comment]
You're right. But when in the perspective of non-developers (designers), it can be confusing maybe? Even, I was not able to understand why the system was not sending email. Is there any error/warning log generation if something like this happens, which I may have missed?
I discovered it when I checked the original function that has been sending e-mail. There, I found at the check for internet-connectivitiy.

However, this is purely an "ease of use" issue. Not that a bug or something.
